.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -11,16 +11,19 @@ an issue where no sections have been filled will be deleted without comment.
 -->
 
 **Is your feature request related to a particular plugin or theme? If so, specify it.**
-The name of the plugin or theme that you would like us to improve. [...]
+The name of the plugin, theme or alias that you would like us to improve. [...]
```

Comment:
Why `alias` specifically?

Reply:
I added alias since some of them need to be changed over time. Specifically git (master -> main) Since that is becoming a change. So if a user wants us to change that in the code they can request that.

If you agree with me resolve this conversation. if not I can remove it from the feature_request.